components/brave_wallet/browser/zcash/zcash_orchard_storage.cc

```diff
@@ -120,9 +162,40 @@ bool ZCashOrchardStorage::CreateSchema() {
                            " ("
                            "account_id TEXT NOT NULL PRIMARY KEY,"
                            "account_birthday INTEGER NOT NULL,"
-                           "latest_scanned_block INTEGER NOT NULL,"
-                           "latest_scanned_block_hash TEXT NOT NULL);") &&
+                           "latest_scanned_block INTEGER,"
+                           "latest_scanned_block_hash TEXT);") &&
+         database_.Execute(
```

Comment:
I think I asked this when we first added this, but can't remember where we landed. Are we encrypting this database?

Reply:
No we don't